### ISSUE TITLE
On Windows, emd gen fails with error "Invalid working directory"

### ISSUE THREAD

**Issue by solvingj** [mh-cbon/emd#26]
```
C:\Users\me\git-c\go-github-release-test>emd gen -in README.e.md -out README.md
2017/06/10 21:13:13 command "gen" failed: Invalid working directory "C:\\Users\\me\\git-c\\go-github-release-test"

```
Seems very likely to be something with the way you enumerate directories not working well on windows.

**Comment by mh-cbon** [comment]
Im afraid in that current situation your setup is unexpected.

In go we have a workspace path $GOPATH, under which there is a special structure to respect,
- src
- bin
- pkg

Its quiet important to follow that, and the path you given does not.

Now assuming, you are not building a go project, that is blocking, 
and indeed, in that perspective, did not make sense.

Ultimately, this path is used to compute some values automatically, see
https://github.com/mh-cbon/emd/blob/e18db0806fa6200024ba0d67f2e6d3a8013dd8fb/main.go#L220

Its definitely possible to make that error non terminal, and add some logic to detect different project layout and so try to find about those variables automatically anyway.

To take an example in a nodejs project that d provided by the package json file, easy, decode&read.
For a java/C/other project, i do not have a clue.

**Comment by solvingj** [comment]
It seems like auto-discovering folder structure for various projects types would be infeasible. Perhaps yet another Yaml file just for emd, for defining whatever paths are needed would be the only way to handle it.

**Comment by solvingj** [comment]
I just tried running this in the go-bin-rpm project and got the same issue.  This command exists in the .version.sh script of go-bin-rpm, so presumably it worked for you?  Although, that file was messed up in several ways, so maybe not.  It looked like string replace issue to me. 

In any case, I notice that go-bin-rpm does not have any of the folders you mentioned. Can you please explain?

**Comment by mh-cbon** [comment]
next release (0.0.13) won t panic anymore, but, that does not solve the question of project layout detection. As alternative, it is possible to enforce values in the prelude of the `e.md` file (https://github.com/mh-cbon/emd#define-data).

____________

> I just tried running this in the go-bin-rpm project and got the same issue.

must be an issue in the location of the package in your the computer you use to gen the readme.

**Comment by mh-cbon** [comment]
Note, i decided to move on 1.0.0/major rather than 0.0.13/minor because i felt like this change might be problematic, moving to a major release makes it clear about that.

**Comment by solvingj** [comment]
yes, this was resolved in 1.0.0
